news/d8390fd1-8c20-4ef8-bea3-ad6b2eb5b3a5.trivial.rst

```diff
@@ -0,0 +1 @@
+Upgrade distlib to 0.3.3
```

Comment:
This file should use `vendor` for the news segment, not `trivial`.

Reply:
Thanks. I've renamed the news segment file